.github/workflows/livetest.yml.disabled

```diff
@@ -0,0 +1,89 @@
+name: Live test
```

Comment:
Let's add a comment what this does and what is needed to enable it (run in cockpit-project org, or you need to set up your own relay host and adjust the `ssh` command in the "Forward cockpit port 9090" step)

Reply:
I will make the RELAY host an env variable and add the comment.